nuitka/build/static_src/CompiledCodeHelpers.c

```diff
@@ -541,6 +541,50 @@ PyObject *BUILTIN_LEN(PyObject *value) {
     return PyInt_FromSsize_t(res);
 }
 
+PyObject *BUILTIN_ANY(PyObject *value) {
+    CHECK_OBJECT(value);
+
+    PyObject *it = PyObject_GetIter(value);
+    PyObject *(*iternext)(PyObject *);
+    PyObject *item;
+
+    if (unlikely((it == NULL)))
+        return NULL;
+    iternext = *Py_TYPE(it)->tp_iternext;
+
+    int cmp;
+
+    for (;;) {
+        item = iternext(it);
+        if (unlikely((item == NULL)))
+            break;
+        cmp = PyObject_IsTrue(item);
+        Py_DECREF(item);
+        if (unlikely(cmp < 0)) {
+            Py_DECREF(it);
+            return NULL;
+        }
+        if (cmp > 0) {
+            Py_DECREF(it);
+            Py_INCREF(Py_True);
+            return Py_True;
+        } else {
```

Comment:
@kayhayen This fixes `print(any([0] * 20000))` tests but breaks `S = [40, 60, 30]
print(any(x > 42 for x in S))`